SequenceAnalysis/src/org/labkey/sequenceanalysis/query/GenbankDisplayColumnFactory.java

```diff
@@ -24,26 +24,25 @@ public GenbankDisplayColumnFactory()
     @Override
     public DisplayColumn createRenderer(ColumnInfo colInfo)
     {
-        DataColumn ret = new DataColumn(colInfo)
+        return new DataColumn(colInfo)
         {
             @Override
-            public void renderGridCellContents(RenderContext ctx, Writer oldWriter, HtmlWriter out) throws IOException
+            public void renderGridCellContents(RenderContext ctx, HtmlWriter out)
             {
-                Object val = ctx.get(getBoundColumn().getFieldKey());
+                String val = ctx.get(getBoundColumn().getFieldKey(), String.class);
                 if (val != null)
                 {
-                    String[] vals = String.valueOf(val).replaceAll("\\s+", "").split(";|,");
+                    String[] vals = val.replaceAll("\\s+", "").split("[;,]");
                     String delim = "";
                     for (String v : vals)
                     {
-                        oldWriter.write(delim + "<a href=" + getFormattedURL(v) + ">" + PageFlowUtil.encode(v) + "</a>");
+                        out.write(delim);
+                        out.write(PageFlowUtil.link(v).href(getFormattedURL(v)));
```

Comment:
This works, but it changes the style from a simple link (previous <a> tag) to a LabKey-styled link (bolded small caps with triangle arrow). If you want the same style, you can add `.clearClasses()` to the end.

Reply:
Is there a different link builder type of class?

Reply:
Nope, this is it. I didn't use PageFlowUtil.link() myself because I forgot we had it. I just called new LinkBuilder() directly. I wish LinkBuilder didn't add the LabKey class by default, but probably too late to change that. Maybe I'll add some static factory methods... link `LinkBuilder.simple(String, URLHelper)`.